libs/coin-modules/coin-aptos/src/exchange/index.ts

```diff
@@ -0,0 +1,16 @@
+import BIPPath from "bip32-path";
+
+export const pathToBuffer = (originalPath: string): Buffer => {
```

Comment:
Doesn't look specific to coin-aptos, if it's really needed it needs to move

Reply:
It might be specific to Aptos given that converting derivation path to buffer yields a different result from `bip32asBuffer`